hack/common.sh

```diff
@@ -41,24 +41,29 @@ readonly OS_SCRATCH_IMAGE_COMPILE_TARGETS=(
   examples/deployment
 )
 readonly OS_IMAGE_COMPILE_BINARIES=("${OS_SCRATCH_IMAGE_COMPILE_TARGETS[@]##*/}" "${OS_IMAGE_COMPILE_TARGETS[@]##*/}")
-
 readonly OS_CROSS_COMPILE_PLATFORMS=(
   linux/amd64
   darwin/amd64
   windows/amd64
   linux/386
 )
-readonly OS_CROSS_COMPILE_TARGETS=(
+
+if [ "_$@" = "_test/e2e/e2e.test" ]; then
```

Comment:
Actually, come to think of it, @pmorie I distinctly remember Jordan mentioning that he didn't want it as a default build product.   So, I think this is the right implementation based on that requirement.......